scripts/west_commands/runners/bossac.py

```diff
@@ -25,6 +25,9 @@
 DEFAULT_BOSSAC_PORT = '/dev/ttyACM0'
 DEFAULT_BOSSAC_SPEED = '115200'
 
+if platform.system() == 'Darwin':
+    DEFAULT_BOSSAC_PORT = '/dev/cu.usbmodem14301'
```

Comment:
Just a style thing, but can you put the other setting underneath as:

```py
else:
    DEFAULT_BOSSAC_PORT = '/dev/ttyACM0'
```

Reply:
Yep! I've amended the commit, I moved this conditional up to where the constant was originally set also.

Reply:
Thanks!

One more question before I approve: can you confirm that '/dev/cu.usbmodem14301' is the same everywhere? I'm confused about the '14301' suffix at the end of the device file name. It looks like a randomly generated or device-specific string and I just want to make sure that all devices actually have the same file name.

For comparison, `/dev/ttyACM0` on Linux is always the first TTY ACM device which was plugged into USB, so it's a safe default.

Reply:
Yes, I'm not so sure about that now. I was trying to find out where I'd read it from, where the number after `usbmodem` is generated somehow from the VID and PID, but I can't find any evidence of that. I think it's probably best if I remove this for now. In theory, the proper way of identifying the port is taking the device node from `ioreg`.

Reply:
Another option is to rework this to prompt the user if there are multiple candidates on each OS. See get_board_snr() in the nrfjprog.py runner file for an example.

Reply:
OK, should be resolved with this next commit!